### PR TITLE
Validation: Fix duplication description metadata

### PR DESCRIPTION
**Pull request by MeeraDi**
This PR address duplication descriptions in article metadata. 
This PR does NOT resolve build issues related to duplicate H1, duplicate titles, and invalid tab groups.

### CODE REVIEW

File: api-reference/beta/api/security-ediscoverycustodian-post-sitesources.md [hunk 1]
```diff
@@ -1,13 +1,13 @@
 ---
-title: "Create custodian siteSource"
-description: "Create a new custodian siteSource object."
+title: Create custodian siteSource (microsoft.graph.security)
```
Comment (Lauragra) [inline api-reference/beta/api/security-ediscoverycustodian-post-sitesources.md:2]: Global: Please do not add the namespace to disambiguate the topic titles. We have a PR to deprecate the APIs in the ediscovery namespace, and those topics will eventually be removed. Meanwhile, adding the namespace to titles is not a consistent approach. We will be adding (deprecated) to the titles of the duplicated reference topics.

Comment (MeeraDi) [reply api-reference/beta/api/security-ediscoverycustodian-post-sitesources.md:2]: Hi @Lauragra I can make this correction and pass on to my team. It looks like in many cases we can resolve the duplication issue but I know in this one I struggled. Can you suggest an alternative to disambiguate? I don't think these PRs will merge with outstanding warnings. 
CC: @v-emilypr for team visibility.

**Comment by MeeraDi** [comment]
**Global effort to fix build validation errors**
@Lauragra @Danielabom 

The Content & Learning team is fixing build validation errors on docs.microsoft.com for the rest of H2. This will eliminate potential accessibility, security, and usability issues. This PR includes only build validation fixes and does not change other content. Please review within five business days and merge or comment in the PR with any changes you'd like to see. Thank you!

**Comment by MeeraDi** [comment]
Closing, the repo now allows duplicate descriptions.

**Pull request closed without merging**